src/bin.mts

```diff
@@ -45,6 +46,20 @@ const args = yargs(process.argv)
     description: 'Override the VS Code version used to run tests',
     group: vscodeSection,
   })
+  .option('install-extensions', {
+    alias: 'e',
```

Comment:
Let's not have a short alias for this, as this is not likely to be a common option

Reply:
Agreed, should mostly be driven by dependentExtensions 99% of the time with the occasional override for a test/nightly .vsix

Reply:
Fixed 68c968f